Lunch/LoginViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  LoginViewController.swift
+//  Lunch
+//
+//  Created by Spencer Amann on 8/1/15.
+//  Copyright (c) 2015 Spencer Amann. All rights reserved.
+//
+
+import UIKit
+import Parse
+
+class LoginViewController: UIViewController {
+
+    @IBOutlet weak var userNameTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+
+    let loginIdentifier = "loginWithUsername"
+    let createAccountIdentififer = "createAccount"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationItem.hidesBackButton = true
+    }
+
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        if let currentUser = PFUser.currentUser() {
+            if let username = currentUser.username {
+                self.performSegueWithIdentifier(self.loginIdentifier, sender: self)
+            }
+        }
+    }
+
+    @IBAction func loginButtonClick(sender: AnyObject) {
+        if let username = userNameTextField.text, password = passwordTextField.text {
+            PFUser.logInWithUsernameInBackground(username, password: password, block: { (user: PFUser?, error:NSError?) -> Void in
+                if user != nil && error == nil {
+                    self.performSegueWithIdentifier(self.loginIdentifier, sender: self)
+                } else {
+                    let notPermitted = UIAlertView(title: "Error", message: "Login Failed. \nWhoops!", delegate: nil, cancelButtonTitle: "OK")
+                    notPermitted.show()
```

Comment:
That's a weird variable name. "show not permitted" doesn't really make any sense. I usually just call it `alertView` lol